modules/fluid_properties/include/userobjects/HEMFluidProperties.h

```diff
@@ -0,0 +1,193 @@
+#ifndef HEMFLUIDPROPERTIES_H
```

Comment:
Need the modules header here and throughout.

Reply:
Done.